enterprise/internal/batches/background/workers_test.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/sourcegraph/sourcegraph/enterprise/internal/batches/store"
 	ct "github.com/sourcegraph/sourcegraph/enterprise/internal/batches/testing"
-	"github.com/sourcegraph/sourcegraph/internal/batches"
+	btypes "github.com/sourcegraph/sourcegraph/enterprise/internal/batches/types"
```

Comment:
How about... ?

```suggestion
	ty "github.com/sourcegraph/sourcegraph/enterprise/internal/batches/types"
```

(It's the most common short variable name in compilers/typecheckers when referring to types)

Reply:
Is this common in go, or just compilers in general? For the singular case I mostly see `typ` because `type` is a reserved keyword
Happy to change, just want to make sure to the right value 😬

Reply:
`typ` is also fine (as you know, in German it also works: "wasn typ ey")

Reply:
Not a blocker, feel free to ignore!

Reply:
I tinkered around with it but didn't like the phonetics of `ty.BatchChange` and friends too much, feel free to make more suggestions or try to convince me and I'll change it :)

Reply:
No, that's fine!

(Of course, the _other_ option would be to not have a single package. See "Don't use a single package for all your APIs" here: https://blog.golang.org/package-names. Not saying you should do this, but it could be interesting to see what a `changeset` package would look like, with a `changeset.ReconcilerState` or `changeset.ExternalState` etc.)